apps/cookbook/src/app/examples/item-example/examples/ellipsis-behavior.ts

```diff
@@ -0,0 +1,29 @@
+import { Component } from '@angular/core';
+import { ItemModule } from '@kirbydesign/designsystem/item';
+
+const config = {
+  selector: 'cookbook-item-example-ellipsis-behavior',
+  template: `
+<kirby-item>
+    <p class="kirby-item-title">Fusce id neque suscipit, finibus urna convallis, auctor arcu.</p>
+    <data class="kirby-item-detail" slot="end">22.86%</data> 
+</kirby-item>
+
+<kirby-item>
+    <kirby-label direction="horizontal">
+        <p class="kirby-item-title" >Fusce id neque suscipit, finibus urna convallis, auctor arcu.</p>
+        <data class="kirby-item-detail">22.86%</data> 
+    </kirby-label>
+</kirby-item>
```

Comment:
We usually only add examples that are also shown in the showcase, but since this example exists for testing purposes I think it is a better fit for the item component stories: https://github.com/kirbydesign/designsystem/blob/develop/libs/designsystem/item/src/item.component.stories.ts

So my suggestion is to remove this example and add the template as a new story in that file ☝️